next.clayui.com/content/docs/get-started/components-status.md

```diff
@@ -1,66 +0,0 @@
----
```

Comment:
@jbalsas just out of curiosity, why are we removing this?

Reply:
this is duplicated and already live at https://github.com/liferay/clay/blob/master/next.clayui.com/content/docs/get-started/how-to-read-this-documentation.md, isn’t it?

Reply:
Oh yeah, I was assuming we had just renamed the file name 😅. Thanks @jbalsas